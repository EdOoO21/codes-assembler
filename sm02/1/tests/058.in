-597 -579 512 314 938 -589 -916 -606 -80 -170 -277 932 -411 649 795 -467 979 -79 646 260 -142 -419 785 187 -786 -975 2 -563 -38 544 -850 -74 133 351 -229 351 -247 -874 -793 604 366 592 -725 -382 -930 -623 777 -399 -365 -648 253 239 486 -503 559 392 870 -846 -675 -588 -82 -744 -734 -732 472 802 -535 -938 -461 699 -487 376 472 -797
