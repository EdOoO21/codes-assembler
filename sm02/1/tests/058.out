-797
472
376
-487
699
-461
-938
-535
802
472
-732
-734
-744
-82
-588
-675
-846
870
392
559
-503
486
239
253
-648
-365
-399
777
-623
-930
-382
-725
592
366
604
-793
-874
-247
351
-229
351
133
-74
-850
544
-38
-563
2
-975
-786
187
785
-419
-142
260
646
-79
979
-467
795
649
-411
932
-277
-170
-80
-606
-916
-589
938
314
512
-579
-597
